examples/librimix/librimix.ipynb

```diff
@@ -0,0 +1,771 @@
+      "text/plain": [
+       "<Figure size 1152x144 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.matshow(sample['sources'][0].T);"
```

Comment:
dunno whether there is a way to show them upside down so they will look more familiar?
The .T (transpose?) is not a common notation I've seen; I think .transpose() or .transpose(0,1) if necessary, would be easier to read, no?

Reply:
Yeah, I'll flip them. Yes, .T stands for "transpose". I don't mind using the longer variant.